src/css/curCSS.js

```diff
@@ -1,16 +1,24 @@
 import jQuery from "../core.js";
 import isAttached from "../core/isAttached.js";
 import getStyles from "./var/getStyles.js";
+import rcustomProp from "./var/rcustomProp.js";
+import rtrim from "../var/rtrim.js";
 
 function curCSS( elem, name, computed ) {
-	var ret;
+	var ret,
+		isCustomProp = rcustomProp.test( name );
 
 	computed = computed || getStyles( elem );
 
 	// getPropertyValue is needed for `.css('--customProperty')` (gh-3144)
 	if ( computed ) {
 		ret = computed.getPropertyValue( name ) || computed[ name ];
 
+		// trim whitespace for custom property (issue #4926)
```

Comment:
That's a jQuery-specific gotcha:
```suggestion
		// trim whitespace for custom property (issue gh-4926)
```
Since we have lots of references using `#` to our legacy Trac bug tracker at https://bugs.jquery.com/, we use the `gh-XXXX` syntax to make it explicit what comes from GitHub.

We'll need to replace these legacy `#` uses one day to reduce confusion.